core/Utils.py

```diff
@@ -94,3 +96,37 @@ def print_yaml_nicely(cls, to_print):
         """
         import json
         print json.dumps(to_print, indent=2)
+
+"""
+Used for re-typing variables
```

Comment:
There is some logic for french and some logic for arithmetic. For example, with divisions, we need to cast one of the operators to float in order to get a get a float result. But for aesthetic reasons we don't want `8.0 / 2 = 2.0` ("deux virgule zéro").
I don't know what your course of action is for i18n but yes, these yes these methods will return "virgule" in all locales.
Also, it was trial and error. Spoken "deux" through STT sometimes results in "deux" and sometimes results in "de". It might be judicious (as you suggest) to move specific Utils to the Neuron.